uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt

```diff
@@ -295,7 +295,7 @@ private fun FeedList(
     onFavoriteChange: (FeedItem) -> Unit,
     onClickPlayPodcastButton: (FeedItem.Podcast) -> Unit,
     listState: LazyListState,
-    showItemCount: Boolean,
+    isFilterState: Boolean,
```

Comment:
Reason: Because I want to use this variable not only to show filtered item count, but also to check whether FirstFeedItem is filtered or not, I make variable-name more abstract.